### ISSUE TITLE
Commit property size too small in PostGres

### ISSUE THREAD

**Issue by gmadkat** [javers/javers#462]
Can we have the commit property value to be larger than 200? we need it at least to be 1024

```
CREATE TABLE dave.jv_commit_property
(
  property_name character varying(200) NOT NULL,
  property_value character varying(200),
  commit_fk bigint NOT NULL,
  CONSTRAINT jv_commit_property_pk PRIMARY KEY (commit_fk, property_name),
  CONSTRAINT jv_commit_property_commit_fk FOREIGN KEY (commit_fk)
      REFERENCES dave.jv_commit (commit_pk) MATCH SIMPLE
      ON UPDATE NO ACTION ON DELETE NO ACTION
)

```

**Comment by gmadkat** [comment]
I am happy to make any changes and submit a pull request if you are ok accepting them.

**Comment by bartoszwalacik** [comment]
I dont think it is a goog idea. Commit properties are designed as short tags, useful for querying and not for storing large data. What kind of data you want to store as commit property ?

**Comment by gmadkat** [comment]
We have a requirement where we are porting legacy comments which are over 512 characters, we might be able to live with 512 or 600 but definitely 200 is too short for us. 
I am guessing there is no size limit for Mongo? Only in relational databases?

**Comment by bartoszwalacik** [comment]
yes, there is no limits in Monog. As a quick workaround, simply run `ALTER COLUMN` on this column

**Comment by gmadkat** [comment]
Thank you, do you know when you can fix the size and if so, what is the new size you will use?

**Comment by bartoszwalacik** [comment]
seems not very urgent as you have workaround. I will set size to 600 as you suggest. It will be released with the next release

**Comment by bartoszwalacik** [comment]
fixed in 2.7.2
